### ISSUE TITLE
"I can't find any complete commands to process!"

### ISSUE THREAD

**Issue by jonsterling** [ProofGeneral/PG#179]
I'm having a strange issue; with a completely fresh install of PG and Coq (8.6), almost every time I try to check a development in Emacs I receive the error `"I can't find any complete commands to process!"`

For the record, this development parses and compiles fine with Coq. Has this happened to anyone else? I've tried it both with `company-coq` enabled and disabled, in case there was some weird interaction between the packages.

Thanks for any help you can give!

**Comment by cpitclaudel** [comment]
An example would help. Does this actually happen with any script?

**Comment by jonsterling** [comment]
@cpitclaudel Yes, as far as I can tell it happens for any script, even a script of just one line. I'm trying to determine at the moment whether this is being caused by other packages that are loaded, since PG works fine for all my friends...

**Comment by jonsterling** [comment]
@cpitclaudel OK, I think I have determined the problem (at least at a superficial level). When I disable the package [`pollen-mode`](https://github.com/lijunsong/pollen-mode) in my Emacs configuration, PG works fine. I bet the bug is in `pollen-mode`, not PG...

For now, I'll just disable that mode; once I learn more about Emacs hacking, maybe I'll be able to understand what is really going on.

I think you can feel free to close this ticket if you don't think there's anything to be done on PG's side.

**Comment by cpitclaudel** [comment]
Hmm, surprising.  Why should pollen-mode run while PG is active ? Or are you editing Coq snippets inside a pollen file ?

**Comment by jonsterling** [comment]
@cpitclaudel It's so weird! `pollen-mode` is not activated, but somehow just having the package loaded in my config causes a problem, as far as I can tell. I wish I could be more precise about what is happening, but I am just a "user" of Emacs and don't really understand how it works yet.

**Comment by cpitclaudel** [comment]
How do you load the package ?

**Comment by jonsterling** [comment]
I am using [`use-package`](https://github.com/jwiegley/use-package).

**Comment by cpitclaudel** [comment]
Can you show the full use-package form that you're using ?

**Comment by jonsterling** [comment]
@cpitclaudel Here's what I had:

```elisp
(use-package pollen-mode
  :ensure t)
```

I loaded PG later in the file in the following way:

```elisp
(let ((proof-site "~/.emacs.d/PG/generic/proof-site.el"))
  (when (file-exists-p proof-site)
    (require 'proof-site proof-site)
    (use-package company-coq
      :ensure t
      :config (add-hook 'coq-mode-hook #'company-coq-mode))))
```

**Comment by cpitclaudel** [comment]
Got it, thanks. I'll try to reproduce this.
